components/Buttons.js

```diff
@@ -9,6 +10,17 @@ export function ButtonsForm() {
   );
 }
 
+export function ButtonsEdit() {
+  return (
+    <StyledButtonContainer>
+      <button type="submit">Update</button>
+      <Link passHref href="/characters">
+        <button>Cancel</button>
```

Comment:
You should use an anchor tag inside the Link component, as stated in the documentation:
https://nextjs.org/docs/api-reference/next/link